superset/assets/src/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -427,31 +427,15 @@ export default class AnnotationLayer extends React.PureComponent {
             <div style={{ marginTop: '1rem' }}>
               <CheckboxControl
                 hovered
-                name="annotation-override-since"
-                label="Override 'Since'"
-                description={`This controls whether the "Since" field from the current
+                name="annotation-override-time_range"
+                label="Override time range"
+                description={`This controls whether the "time_range" field from the current
                   view should be passed down to the chart containing the annotation data.`}
-                value={!!Object.keys(overrides).find(x => x === 'since')}
+                value={!!Object.keys(overrides).find(x => x === 'time_range' || x === 'since' || x === 'until')}
```

Comment:
this line is to backward compatible with old override options: since or until. now we only allow override whole time_range, so either since or until will be treated like override time_range.